yt_dlp/extractor/twitcasting.py

```diff
@@ -248,16 +248,23 @@ def _real_extract(self, url):
             'Pass "https://twitcasting.tv/{0}/show" to download the history'.format(uploader_id))
 
         webpage = self._download_webpage(url, uploader_id)
+        is_live = 'data-is-onlive="true"' in webpage
+        if not is_live:
+            raise UserNotLive(video_id=uploader_id)
```

Comment:
```suggestion
        if 'data-is-onlive="true"' not in webpage:
            raise UserNotLive(video_id=uploader_id)
```
so we want to skip checking the live history page in this case?

Reply:
Ah, live streams with passwords don't seem to show up, so it's better to delete it.